### ISSUE TITLE
Feature request: Check if docker image is built before running

### ISSUE THREAD

**Issue by paulrobinette** [duckietown/duckietown-shell#85]
Is there a way to check if a dts container is built before it is run? I have students who use the class repo template, clone it, then try to run it and it gives an error about permission denied on docker (because the image doesn't exist on docker). It would be nice to have a friendly "Have you built your container using dts devel build yet?" message at the end as a prompt. Preferably in bright colors so they see it!

**Comment by liampaull** [comment]
This is for the `dts devel run` command correct?

**Comment by paulrobinette** [comment]
Correct. The current error message makes sense if you know docker but
confuses new students in my class. A quick, friendly reminder to do dts
devel build first would be appreciated.
